lib/hashie/extensions/mash/symbolize_keys.rb

```diff
@@ -24,13 +24,13 @@ def self.included(base)
 
         private
 
-        # Converts a key to a symbol
+        # Converts a key to a symbol, if possible
         #
         # @api private
-        # @param [String, Symbol] key the key to convert to a symbol
-        # @return [void]
+        # @param [<K>] key the key to attempt convert to a symbol
+        # @return [Symbol, K]
```

Comment:
I'm not sure if there is a convention or syntax for a generic type variable in Yard... I mean to say that the return value is one of Symbol or specifically the type of the input param...

Reply:
I don't know, LGTM.